events/sql.py

```diff
@@ -1,11 +1,12 @@
 from django.db import connection
 
 
-def count_events_for_keywords(keyword_ids=()):
+def count_events_for_keywords(keyword_ids=(), all=False):
     """
     Get the actual count of events using the given keywords.
 
-    :param keyword_ids: set of keyword ids; pass an empty set to get the data for all keywords
+    :param keyword_ids: set of keyword ids
+    :param all: count all keywords instead
```

Comment:
This should be after `:type keyword_ids:` (and it's missing a `:type:` clause too).